# !TOPICS
# clock_topic: "/clock"
# cpu_topic: "/cpu_monitor/smf_move_base_planner/cpu"
# goal_reached_topic: "/smf_move_base_planner/goal_reached"
# goal_topic: "/goal_available"
# collision_counter_topic: "/collision_counter"
# num_nodes_topic: "/smf_move_base_planner/smf_num_nodes" # only considered if the approach is sampling based
# agent_states_topic: "/pedsim_simulator/simulated_agents"
# odom_topic: "/pepper/odom_groundtruth"

clock_topic: "/clock" # WORKS
cpu_topic: "/cpu_monitor/smf_move_base_planner/cpu" # UNAVAILABLE
goal_reached_topic: "/smf_move_base_planner/goal_reached" # UNAVAILABLE
goal_topic: "/goal_available" # UNAVAILABLE
collision_counter_topic: "/collision_counter" # UNAVAILABLE
num_nodes_topic: "/smf_move_base_planner/smf_num_nodes" # only considered if the approach is sampling based
agent_states_topic: "/pedsim_simulator/simulated_agents" # WORKS

odom_topic: "/ground_truth"
acceleration_monitor_topic: "/acceleration_monitor"

# octomap service
octomap_service: /smf_move_base_mapper/get_binary

#! robot and agents params
robot_radius: 0.3
robot_height: 1.0
agent_radius: 0.45
robot_max_velocity: 0.22
agent_max_velocity: 0.5

# ! measuring characteristics
measure_rate: 0.5
sim: True # whether the test is being done in simulation or real experiment

#! saving files config
# csv_dir: "/home/sasm/ros/noetic/system/src/pepper_social_nav_tests/results"
csv_dir: "/home/adam/ros/noetic/system/src/pepper_social_nav_tests/results"
approach_name: "smf_planner"
csv_name: "test.csv"

max_test_time: 500

# Jogplotter
# Runs benchmark of several points
# ./ros/noetic/system/src/social_nav_benchmark/scripts/smf_nav_stack/tests_maker.sh 